### ISSUE TITLE
fritzcollectd no longer maintained

### ISSUE THREAD

**Issue by fetzerch** [fetzerch/fritzcollectd#54]
First of all happy new year!

**TL;DR** I've been pretty quiet in this project and I think it's only fair to officially announce that I'm no longer maintaining *fritzcollectd*.
I'll leave the GitHub project open because there definitively is still interest in Fritzbox monitoring and you're helping out each other a lot in the issue section.

For those interested in more details:

My main motivation of starting the project was my instable ADSL2 connection back then. In the meantime a lot has changed. I've switched to a much more stable VDSL connection and I also moved away from AVM to Unifi hardware. Not because I wasn't satisfied with AVM, but I wanted a more professional setup with VLAN support (and more professional data analytics).

In addition the monitoring world has also changed a lot and the project would require some fundamental changes.

When I wrote *fritzcollectd*, *collectd* still was quite popular for server monitoring and it made sense to base the Fritzbox monitoring on it because it provided a scheduler, a configuration framework and a database abstraction. Back then, I wasn't even using *InfluxDB* but *Graphite*. Today many people here only run *collectd* in order to use *fritzcollectd*. And because setting up *collectd* is not trivial, people execute the whole stack using Docker containers.

These days there are simpler ways to fetch data and store it in *InfluxDB*. One approach would be to write a small script that fetches the data and outputs it as JSON. Then *Telegraf* could be configured to launch the script with the *Exec* plugin.
Another way would be to use a program such as https://github.com/yunity/fritzinfluxdb which runs as a service and directly feeds data into InfluxDB.

**Comment by jobenvil** [comment]
farewell @fetzerch and many thanks for fritzcollectd!

Hopefully somebody wants to retake this exciting project. There is a lot of potential :-)

**Comment by heurekus** [comment]
Many thanks for my side as well!
